esx_service/vmdk_ops.py

```diff
@@ -648,7 +653,9 @@ def disk_attach(vmdk_path, vm):
                      if type(dev) == vim.VirtualDisk and dev.controllerKey ==
                      controller_key])
         # search in 15 slots, with unit_number 7 reserved for scsi controller
-        availSlots = set(range(0, 6) + range(8, 16)) - taken
+        range1 = set(range(0, 7))
```

Comment:
Nope, fails as well,
Python 3.4.0 (default, Apr 11 2014, 13:05:11)
[GCC 4.8.2] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> y=set(range(0, 7) | range(8, 16))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unsupported operand type(s) for |: 'range' and 'range'


On Tue, Jun 28, 2016 at 10:52 AM, Mark Sterin <notifications@github.com>
wrote:

> In esx_service/vmdk_ops.py
> <https://github.com/vmware/docker-volume-vsphere/pull/479#discussion_r68698925>
> :
>
> > @@ -648,7 +653,9 @@ def disk_attach(vmdk_path, vm):
> >                       if type(dev) == vim.VirtualDisk and dev.controllerKey ==
> >                       controller_key])
> >          # search in 15 slots, with unit_number 7 reserved for scsi controller
> > -        availSlots = set(range(0, 6) + range(8, 16)) - taken
> > +        range1 = set(range(0, 7))
>
> OK, still no need for 3 lines
> availSlots = (set(range(0, 7) | range(8, 16)) - taken
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/vmware/docker-volume-vsphere/pull/479/files/618f13068273be9c4251cba120e77e704c061ed4#r68698925>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/APHseAf7a2PLZMaDpMZpcQBt1mvzoGHOks5qQK-FgaJpZM4I4qrt>
> .
>